test/Microsoft.AspNetCore.Mvc.Razor.Host.Test/DictionaryPrefixTestTagHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Host.Test
+{
+    [HtmlTargetElement(Attributes = "prefix-*")]
+    public class DictionaryPrefixTestTagHelper : TagHelper
```

Comment:
My apologies, I noticed that the generated file looks different but I'm not sure how to impact the baseline files nor do I know what it's supposed to look like for Dictionary-based TagHelpers.  They are generated utilizing the provided `MvcRazorHost`.  The `TagHelpersWebsite `is functional with the new TagHelper.